### DISCUSSION TITLE
Change Date Format

### DISCUSSION THREAD

**Discussion by nathanthibert** [intlify/vue-i18n#803] [Q&A]
Is it possible to customize the date format to remove some punctuations or add extra text between date and time? I have read through the docs and haven't uncovered how to do this. Or would it be better to use a 3rd party library like MomentJS to accomplish this?

**Comment by kazupon** [comment]
In your use case, you can use the DatetimeFormat component to customize the formatting.
https://vue-i18n.intlify.dev/guide/essentials/datetime.html#custom-formatting

If it doesn't fit your use case, you can use it with a library-like moment.
